### ISSUE TITLE
[Computer Vision Defect Detection Model] - Train Dataset and Test Dataset Settings

### ISSUE THREAD

**Issue by LeeSooHyeon** [aws-samples/amazon-lookout-for-vision#14]
Hello.

I am currently using a **"Computer Vision Defect Detection Model"**.

I would like to know how to configure the training and test datasets in the model code.

### I have set the **[Hyperparameters]** as follows.

`
HyperParameters={
        'ModelType': 'segmentation',
        'TestInputDataAttributeNames': 'source-ref,anomaly-label-metadata,anomaly-label,anomaly-mask-ref-metadata,anomaly-mask-ref',
        'TrainingInputDataAttributeNames': 'source-ref,anomaly-label-metadata,anomaly-label,anomaly-mask-ref-metadata,anomaly-mask-ref'
    }
`

### And i set **[InputDataConfig]** like this.

`
InputDataConfig=[
        {
            'ChannelName': 'training',
            'DataSource': {
                'S3DataSource': {
                    'S3DataType': 'AugmentedManifestFile',
                    'S3Uri': segmentation_train_s3_path,
                    'S3DataDistributionType': 'ShardedByS3Key',
                    'AttributeNames': [
                        'source-ref',
                        'anomaly-label-metadata',
                        'anomaly-label',
                        'anomaly-mask-ref-metadata',
                        'anomaly-mask-ref'
                    ],
                }
            },
            'CompressionType': 'None',
            'RecordWrapperType': 'RecordIO',
            'InputMode': 'Pipe'
        },
        {
            'ChannelName': 'test',
            'DataSource': {
                'S3DataSource': {
                    'S3DataType': 'AugmentedManifestFile',
                    'S3Uri': segmentation_test_s3_path,
                    'S3DataDistributionType': 'ShardedByS3Key',
                    'AttributeNames': [
                        'source-ref',
                        'anomaly-label-metadata',
                        'anomaly-label',
                        'anomaly-mask-ref-metadata',
                        'anomaly-mask-ref'
                    ],
                }
            },
            'CompressionType': 'None',
            'RecordWrapperType': 'RecordIO',
            'InputMode': 'Pipe'
        },
    ]
`

Does this mean the model is trained **only** with the **"training" channel**, and **the evaluation metrics** are calculated using the **"test" channel**?

How can I verify whether the model internally separates the data into training and test sets?

Additionally, if I provide only a "training" channel, does the model internally split the data into training and test sets, or is it trained solely on the provided data?
